pkg/volume/azure_file/azure_file.go

```diff
@@ -207,7 +209,12 @@ func (b *azureFileMounter) SetUpAt(dir string, fsGroup *int64) error {
 		return err
 	}
 	os.MkdirAll(dir, 0750)
-	source := fmt.Sprintf("//%s.file.core.windows.net/%s", accountName, b.shareName)
+
+	azure, err := getAzureCloud(b.plugin.host.GetCloudProvider())
```

Comment:
Or alternatively, we can keep the cloud provider resolving logic in this PR (by examining current provider), but fallback to the global Azure instance if none Azure cloud provider was found.
In this case, we can provide a unified experience for regional cloud, and we're still able to add the azure cloud provider name parameter (which takes precedence) in future if required.